extensions/rich_text_components/Video/Video.js

```diff
@@ -34,6 +34,8 @@ oppia.directive('oppiaNoninteractiveVideo', [
 
           $scope.videoId = oppiaHtmlEscaper.escapedJsonToObj(
             $attrs.videoIdWithValue);
+          $scope.videoId = $scope.videoId.match(
+            /([A-z]|[0-9]){11}/g);
```

Comment:
Would it be possible for this to match specific YouTube URL templates instead (youtube.com/..., youtu.be/..., etc.)? Also, what happens if no match is found?